src/components/Elements/Modal.js

```diff
@@ -23,7 +24,7 @@ function Modal(props) {
       transitionLeaveTimeout={animation.duration.standard}
     >
       { show &&
-        <div key="modal" className="modal" onClick={() => close()}>
+        <div key="modal" className={`modal ${className}`} onClick={() => close()}>
```

Comment:
We could use the classnames lib for this for consistency?

`cx('modal', ...className)`